statistics/handle/handle.go

```diff
@@ -265,6 +265,12 @@ func (h *Handle) Update(is infoschema.InfoSchema) error {
 			deletedTableIDs = append(deletedTableIDs, physicalID)
 			continue
 		}
+		if oldTbl, ok := oldCache.tables[physicalID]; ok {
+			if oldTbl.Version >= version {
+				tables = append(tables, oldTbl)
```

Comment:
This line seems not needed because the stats of this table is not changed.
It would still keep unchanged if we don't append here.